server/startup/migrations/v231.ts

```diff
@@ -0,0 +1,53 @@
+import { ISectionBlock, BlockType } from '@rocket.chat/apps-engine/definition/uikit/blocks/Blocks';
+import { TextObjectType } from '@rocket.chat/apps-engine/definition/uikit/blocks/Objects';
+
+import { Migrations } from '../../../app/migrations/server';
+import { BannerPlatform } from '../../../definition/IBanner';
+import { Banner } from '../../sdk';
+import { settings } from '../../../app/settings/server';
+import { Settings } from '../../../app/models/server';
+
+Migrations.add({
+	version: 231,
+	up() {
+		const LDAPEnabled = settings.get('LDAP_Enable');
+		const SAMLEnabled = settings.get('SAML_Custom_Default');
+		const CustomOauthEnabled = Settings.isOAuthEnabled();
```

Comment:
if this query will be used only on this migration, I think you write the query directly, instead perpetuating it on the model class.